src/sentry/snuba/metrics/query_builder.py

```diff
@@ -724,16 +759,26 @@ def translate_result_groups(self):
                     for data in subresults[k]["data"]:
                         self._extract_data(data, groups)
 
+        group_by_alias_to_group_by_column = (
```

Comment:
```suggestion
        groupby_alias_to_groupby_column = (
```
in another places you are useing without `_`.

Reply:
Sure